wiki/Beatmap_Discussion/en.md

```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions or problems.**
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on Suggestions or Problems.**
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
```

Comment:
isn't it "mapper"

Reply:
nope, creator
at some point, ppy said to use "creator" instead of "mapper"

Reply:
🤔 
![screen shot 2017-12-28 at 11 45 07 pm](https://user-images.githubusercontent.com/30382015/34415352-2b2d99b4-ec29-11e7-8307-d066258e8f3d.png)